src/freenas/usr/local/lib/middlewared_truenas/alert/source/mca_and_apei_events.py

```diff
@@ -0,0 +1,37 @@
+from datetime import timedelta
+
+from middlewared.alert.base import AlertSource
+from middlewared.alert.schedule import IntervalSchedule
+
+
+class MCAAlertSource(AlertSource):
+    products = ('ENTERPRISE',)
+    schedule = IntervalSchedule(timedelta(hours=24))
+
+    async def check(self):
```

Comment:
Is this desired to be posted for the same system every 24 hours?

Should we just return a normal alert and the just hide it from being displayed? (it'll be a new code)

Reply:
> Is this desired to be posted for the same system every 24 hours?
Yes, it was requested by the support team.

> Should we just return a normal alert and the just hide it from being displayed? (it'll be a new code)
Support and PM team do not want to raise these types of alerts to the end-user since there are some of them that are not necessarily fatal.

Reply:
I think we should instead just start a coroutine loop with `sleep(86400)` like we do for usage stats.

Creating an alert has a side effect of this (never emitted) alert appearing in alert configuration dialogs.

Reply:
@themylogin yep, agree 100%. I've completely forgot about @periodic decorator 😬 I'll fix